koku/api/resource_types/aws_org_unit/view.py

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2020 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""View for AWS accounts."""
+from django.db.models import F
+from django.utils.decorators import method_decorator
+from django.views.decorators.vary import vary_on_headers
+from rest_framework import generics
+
+from api.common import CACHE_RH_IDENTITY_HEADER
+from api.common.permissions.resource_type_access import ResourceTypeAccessPermission
+from api.resource_types.serializers import ResourceTypeSerializer
+from reporting.provider.aws.models import AWSOrganizationalUnit
+
+
+class AWSOrganizationalUnitView(generics.ListAPIView):
+    """API GET list view for AWS organizational units."""
+
+    queryset = (
+        AWSOrganizationalUnit.objects.filter(deleted_timestamp__isnull=True)
```

Comment:
@myersCody @abaiken Does this query seem okay to get the list of existing active org units?

Reply:
@lrlevine6 @adberglund 

I am not quite sure if you need a `schema_context`; however, this query looks like it is correctly filtering out the old values.

```
>>> with schema_context("acct10001"):
...     AWSOrganizationalUnit.objects.filter(deleted_timestamp__isnull=True).annotate(**{"value": F("org_unit_id")}).values("value").distinct()
...
<QuerySet [{'value': 'OU_001'}, {'value': 'OU_002'}, {'value': 'OU_003'}, {'value': 'OU_005'}, {'value': 'R_001'}]>`
```

Awesome work Lauren! I know filtering out both the moved & deleted org units can be confusing.